packages/block-library/src/legacy-widget/edit/index.js

```diff
@@ -77,6 +90,7 @@ function NotEmpty( {
 	setAttributes,
 	clientId,
 	isSelected,
+	isWide,
```

Comment:
Will need to default this to `false` if you remove the `isWide={ isWide }` per my other comment.